ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -302,7 +303,8 @@ private KsqlEntity executeStatement(
 
 
   private boolean isExecutableDdlStatement(final Statement statement) {
-    return statement instanceof DdlStatement && !(statement instanceof SetProperty);
+    return statement instanceof DdlStatement
```

Comment:
Consider having a tag interface e.g. `ExecutableStatement` and then have `DdlStatement`, `SetProperty` and `UnsetProperty` extend `ExecutableStatement`. Then this method just becomes `statement instanceof ExecutableStatement` or can indeed be inlined.

Reply:
Ah, yeah, miss read it. Nice one.